common/big_messaging/src/test/scala/uk/ac/wellcome/bigmessaging/fixtures/BigMessagingFixture.scala

```diff
@@ -114,4 +121,23 @@ trait BigMessagingFixture
           )
       }
     }.toList
+
+  def createBrokenPutMemoryTypedStore[T]()(implicit codecT: Codec[T]) = {
```

Comment:
How is it broken out of interest? :slightly_smiling_face:

Reply:
It's only the `.put` method (hence the name) - if you look below it always returns a `Left[StoreWriteError]`.

Reply:
Ah, makes sense

Reply:
Added a comment to that effect.